packages/gatsby-transformer-documentationjs/src/gatsby-node.js

```diff
@@ -426,6 +426,21 @@ exports.onCreateNode = async ({ node, actions, ...helpers }) => {
         })
       }
 
+      // Custom tags
+      if (docsJson.tags && docsJson.tags.length) {
+        // @availableIn, used to specify allowed APIs for restricted actions
+        const availableIn = docsJson.tags.find(
+          tag => tag.title === `availableIn`
+        )
+        if (availableIn) {
+          picked.availableIn = availableIn.description
+            .split(`\n`)[0]
+            .replace(/[[\]]/g, ``)
+            .split(`,`)
+            .map(api => api.trim())
+        }
+      }
+
```

Comment:
could we just add `tags` to `picked` object, and add `availableIn` resolver (that will find custom `availableIn` tag) in just .org? We wouldn't need to add `tags` to field defs (tho it wouldn't hurt I guess)

Reply:
done.
did not add `tags` to the typedefs though - i think we should do a different PR that makes `DocumentationJs` `@dontInfer` and explicitly adds all fields. These are currently missing from the typedefs:

```graphql
commentNumber: Int
deprecated: DocumentationJSComponentDescription
description: DocumentationJSComponentDescription
level: Int
lineNumber: Int
optional: Boolean
tags: [DocumentationJsTags]
title: String
```